src/App.js

```diff
@@ -0,0 +1,15 @@
+import { Routes, Route } from 'react-router-dom';
+import { TextInput, MainPage } from '../src/components';
+import './App.scss';
+
+const App = () => { 
+
+  return (
+    <Routes>
+      <Route path="/tasks" element={<MainPage/>} />
+      <Route path={`/tasks/:id/text`} element={<TextInput/>} />
```

Comment:
Why do you need _/text_ in the end of path?
And for that you need **{** and **`**

Reply:
TextInput - is it a page? Sounds like component. Rename, please